src/Microsoft.CodeAnalysis.Analyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,8 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+RS1034 | MicrosoftCodeAnalysisPerformance | Warning | PreferIsKindAnalyzer
```

Comment:
This shouldn't be here I think, but I'd like to wait to see if the CI is going to fail before removing this line. It should fail IMO.

Reply:
CI is indeed reporting this:

> src/Microsoft.CodeAnalysis.Analyzers/Core/AnalyzerReleases.Unshipped.md(7,1): error RS2006: (NETCORE_ENGINEERING_TELEMETRY=Build) Rule 'RS1034' has duplicate entry between release 'unshipped' and release '3.3.3'